src/slider.js

```diff
@@ -25,7 +25,7 @@ export default class Slider extends React.Component {
   }
 
   // handles responsive breakpoints
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
@tompere I am worried that changing componentWillMount to componentDidMount here might break some functionality